config/environments/production.rb

```diff
@@ -130,12 +130,12 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: protocol + ENV["APP_DOMAIN"].to_s }
   ActionMailer::Base.smtp_settings = {
-    address: "smtp.sendgrid.net",
-    port: "587",
-    authentication: :plain,
-    user_name: "apikey",
-    password: ENV["SENDGRID_API_KEY"],
-    domain: ENV["APP_DOMAIN"],
+    address: ENV["SMTP_ADDRESS"],
```

Comment:
I think we should fallback here to the original default that way when we deploy to already set up Forems won't need new keys immediately and things won't break. Maybe add a comment that they are deprecated and will be removed at some point.

Reply:
yeah that makes sense. I would then also need to add a fallback for everything else here.